src/clusterfuzz/_internal/bot/fuzzers/centipede/engine.py

```diff
@@ -26,20 +26,18 @@
 from clusterfuzz.fuzz import engine
 
 _CLEAN_EXIT_SECS = 10
-_TIMEOUT = 25
 _SERVER_COUNT = 1
 _RSS_LIMIT = 4096
 _RLIMIT_AS = 5120
 _ADDRESS_SPACE_LIMIT = 0
 _DEFAULT_ARGUMENTS = [
     '--exit_on_crash=1',
-    f'--timeout={_TIMEOUT}',
     f'--fork_server={_SERVER_COUNT}',
     f'--rss_limit_mb={_RSS_LIMIT}',
     f'--address_space_limit_mb={_ADDRESS_SPACE_LIMIT}',
 ]
 
-_CRASH_REGEX = re.compile(r'[sS]aving input to:? [\n]?(.*)')
+_CRASH_REGEX = re.compile(r'Saving input to: (.*)')
```

Comment:
Should we keep support for the old format to ensure more compatibility over older builds?

Reply:
I can certainly make it more backward-compatible.
In which case would older builds be used?

Reply:
e.g. when we do our regression (bisection) task. We run older builds to determine which commit range introduced the vulnerability.

Reply:
Ah, I see, thanks!
I will try to make it compatible with all previous build, then : )